src/Databases/DatabaseReplicated.cpp

```diff
@@ -1993,14 +1993,13 @@ DatabaseReplicated::getTablesForBackup(const FilterByNameFunction & filter, cons
 
         StoragePtr storage;
         if (create.uuid != UUIDHelpers::Nil)
-        {
             storage = DatabaseCatalog::instance().tryGetByUUID(create.uuid).second;
-            if (storage)
-                storage->adjustCreateQueryForBackup(create_table_query);
```

Comment:
This call was unnecessary because a consistent metadata snapshot contains table definitions with already applied metadata changes. And this unnecessary call slowed down backups with huge number of tables because `adjustCreateQueryForBackup()` uses ZooKeeper with retries.